### ISSUE TITLE
WCAG compliance

### ISSUE THREAD

**Issue by ubesluttsom** [cybernetisk/internsystem-v2#59]
The website should be tested for [WCAG compliance](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines).
